src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -199,11 +215,18 @@ private static X509Certificate2 CertificateFromThumbprint(X509Store store, strin
             {
                 try
                 {
+                    // We open ReadOnly first to prevent elevation in Windows when running as a non-admin
+                    // Non-admins can read from the LocalMachine store but not write to the store
+
                     store = new X509Store(storeName, storeLocation);
-                    store.Open(OpenFlags.ReadWrite);
+                    store.Open(OpenFlags.ReadOnly);
                     existingCert = CertificateFromThumbprint(store, certificate.Thumbprint);
+                    store.Dispose();
```

Comment:
Recommend 'using' statement instead of explicit Dispose (in case of failure before the Dispose).